Gems/SurfaceData/Code/Source/Editor/EditorSurfaceDataSystemComponent.h

```diff
@@ -22,6 +23,15 @@ namespace AZ::Data
 
 namespace SurfaceData
 {
+    namespace Constants
+    {
+        static const char* s_allTagNames[] = {
+            Constants::s_unassignedTagName,
+            Terrain::Constants::s_terrainHoleTagName,
+            Terrain::Constants::s_terrainTagName,
```

Comment:
These shouldn't need to get added here.  I think they should become available when the terrain surface data component registers itself with the surface data system, right?

Reply:
Removed